docs/integrations/platforms/kubernetes/infisical-push-secret-crd.mdx

```diff
@@ -5,10 +5,23 @@ description: "Learn how to use the InfisicalPushSecret CRD to push and manage se
 ---
 
 
-## Push Secrets to Infisical
+## Overview 
 
+The **InfisicalPushSecret** CRD allows you to create and manage secrets in Infisical and sync them to your Kubernetes cluster as native **Kubernetes secret** resources.
+If you need automatic redeployments of your application when secrets change, you can use the [InfisicalSecret](/docs/integrations/platforms/kubernetes/infisical-secret-crd) CRD to fetch secrets from Infisical and inject them into your application's environment.
 
-### Example usage
+
+This CRD offers the following features:
+- **Create Secrets** in Infisical and manage its lifecycle.
+- **Automatic secret deletion** when the InfisicalPushSecret CRD is deleted.
```

Comment:
For first one, I think we should emphasize that it allows them to take Kubernetes secrets and push them to Infisical.

For the second one, i'd update it to something along the lines of manage the life cycle of pushed secrets and maybe expect on what that means